public/header.php

```diff
@@ -0,0 +1,222 @@
+        padding: 0px;
+        margin: 0px;
+      }
+
+      .menu li {
+        list-style-type: none;
+        padding: 5px;
+        margin: 0;
+        display: inline;
+        background-color: lightgrey;
+        color: white;
+        font-size: 18px;
+        float: left;
+        border: 1px solid grey;
+      }
+
+      .menu li a:hover {
+        background-color: grey;
+        color: white;
+
+      }
+
+
+      #insideactivity, #fullactivity {
+        display: block;
+        float: right;
+        background-color: lightgrey;
+        color: white;
+        position: fixed;
+        bottom: 500px;
+        right: 250px;
+        transform: rotate(270deg);
+        transform-origin: center top 0;
+        padding: 0px 30px 0px 30px;
+        text-align: center;
+        border: solid 1px #c3c3c3;
+      }
+
+      #insideactivity {
+        padding: 25px;
+        display: none;
+        position: fixed;
+        bottom: 500px;
+        right: 250px;
+        transform: rotate(270deg);
+      }
+
+      .clickable {
+        cursor: pointer;
+      }
+
+      table {
+
+      }
+
+      table, th, td {
+        border: 1px solid black;
+        padding: 5px;
+        margin: 5px;
+        width: 450px;
+        height: 25px;
+      }
+
+      table {
+        background-color: lightgrey;
+      }
+
+      th, td {
+        background-color: white;
+      }
+
+      .thread-post {
+        background-color: lightgrey;
+        width: 100%;
+        height: 300px;
+      }
+
+    </style>
+  </head>
+
+  <body>
+
+    <div class="container">
+      <h4>
+        rdripley.com
+      </h4>
+
+
+      <ul class="menu">
+        <li><a href="/Home/default.asp">
+          Home
+          </a>
+        </li>
+
+        <li><a href="/Account/default.asp">
+          Account
+          </a>
+        </li>
+
+        <li>
+
+          <?php
+            $loggedIn = ! empty($_SESSION['user']);
+            if ($loggedIn) {
+              echo "Welcome, " . $_SESSION['user']; ?>
+              <a href="logout.php">
+                Log out
+              </a>
+              <?php
+            } else { ?>
+
+              <div id="flip" class="clickable"> Log-in </div>
+
+              <div id="panel">
+
+                <form action="login.php" method="post">
+
+                  <label>
+                    Username
+                    <input type="text" name="username">
+                  </label>
+
+                  <br>
+
+                  <label>
+                    Password
+                    <input type="password" name="password">
+                  </label>
+
+                  <button type="submit">
+                    Log-in
+                  </button>
+
+                  <br>
+
+                  <a href="register.php">
+                    Sign up now!
+                  </a>
+                </form>
+              </div>
+              <?php
+            }
+          ?>
+
+        </li>
+      </ul>
+
+      <nav>
```

Comment:
I think we should add the activity part of the code to the footer.php file. It is going to be on every page so it will function the same as the header content.